package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@medrunner/api-client",
-  "version": "0.0.1-rc.2",
+  "version": "0.0.1-rc.3",
```

Comment:
since we're only adding dev dependencies and not changing any of the actual library code, it the version bump necessary?

Reply:
Yeah not really

Reply:
Went back to 0.0.1-rc.2 and removed tag v0.0.1-rc.3 in 1289ab2e23a0b185962533363eaa925ef6f50cb1